packages/suite/src/components/suite/Labeling/index.tsx

```diff
@@ -1,7 +1,7 @@
 // "Internal labeling components"
 import AccountLabeling from './components/Account/Container';
 import AddressLabeling from './components/Address/Container';
-import WalletLabeling from './components/Wallet';
+import { WalletLabeling } from './components/Wallet';
```

Comment:
Named exports are inconsistent with our use of default exports here. Their use is more down to preference/opinion but we should definitely prevent inconsistencies to keep the code readable.